packages/strapi-generate-plugin/json/package.json.js

```diff
@@ -43,7 +43,7 @@ module.exports = scope => {
     },
     'dependencies': {},
     'devDependencies': {
-      'strapi-helper-plugin': '3.0.0-alpha.4.8'
+      'strapi-helper-plugin': '3.0.0-alpha.6.3'
```

Comment:
It should more dynamic. Take a look at the [strapi-generate-new](https://github.com/strapi/strapi/blob/master/packages/strapi-generate-new/json/package.json.js#L33) package